### PR TITLE
Adding a mixin & non-Tailwind example for useTransition

### PR DESCRIPTION
**Pull request by weaverryan**
Hi!

I hope this can help the docs. This is all copy-and-pasted from a minimal, functional project - so it definitely works :).

Thank you!

### CODE REVIEW

File: docs/use-transition.md [hunk 1]
```diff
@@ -89,3 +89,74 @@ Here is a typical dropdown component from Tailwind.
...
+            element: this.boxToCloseTarget,
+            enterActive: 'fade-enter-active',
+            enterFrom: 'fade-enter-from',
+            enterTo: 'fade-enter-to',
+            leaveActive: 'fade-leave-active',
+            leaveFrom: 'fade-leave-from',
+            leaveTo: 'fade-leave-to',
+            hiddenClass: 'd-none',
+            // set this, because the item *starts* in an open state
+            transitioned: true,
```
Comment (weaverryan) [inline docs/use-transition.md:129]: This relates to #119

File: docs/use-transition.md [hunk 2]
```diff
@@ -63,7 +63,7 @@ Here is a typical dropdown component from Tailwind.
 <div class="relative"
      data-controller="transition click-outside"
      data-transition-target="content"
-     data-action="click-outside:click:ouside->">
+     data-action="click-outside:click:outside->transition#leave">
```
Comment (weaverryan) [inline docs/use-transition.md:66]: Please let me know if this was *not* a typo - and I am unaware of some magic syntax :)

**Comment by adrienpoly** [comment]
Thanks 👍 

all-contributors please add @weaverryan to doc

**Comment by adrienpoly** [comment]
Thanks 👍

@all-contributors please add @weaverryan to doc

**Pull request merged**